source/threads.tex

```diff
@@ -2472,6 +2599,10 @@
 mutex types\iref{thread.mutex.requirements.mutex} and additionally
 meet the requirements set out below. In this description,
 \tcode{m} denotes an object of a shared mutex type.
+\begin{note}
+The shared mutex types meet the \oldconcept{SharedLockable}
+requirements\iref{thread.req.lockable.shared}.
+\end{note}
 
 \pnum
```

Comment:
Missed edit about "shared time mutex":

    Add a note after 32.5.4.5.1 [thread.sharedtimedmutex.requirements.general] p1:

    [ Note ?: The shared timed mutex types meet the Cpp17SharedTimedLockable requirements (?.?.?.? [thread.req.lockable.shared.timed]). — end note ]

Reply:
@burblebee: agreed, it looks like a note is missing. I can add it during merging, though.

Reply:
Added.